packages/@okta/vuepress-site/docs/guides/registration-inline-hook/activate-and-enable/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Activate and enable
+---
+
+The Registration Inline hook must be set up and activated within your Okta Admin Console and enabled in the self-registration page.
+
+To set up and activate the registration inline hook:
+
+1. Navigate to the **Workflow** > **Inline Hooks** page.
+2. Click **Add Inline Hook** and select **Registration Hook** from the drop-down menu.
+3. Add a name for the hook and your external service URL, including the endpoint (in this example, `registrationHook`).
+4. Include authorization data if required.
+5. Click **Save**.
+
+![Registration Inline Hook set up window](/img/registration-inline-hook-setup.png "Registration Inline Hook set up window")
+
+> **Note:** You can also setup an inline hook using an API. See [Inline Hook Setup](/docs/concepts/inline-hooks/#inline-hook-setup) for further information.
+
+The registation inline hook is now set up with a status of active.
```

Comment:
Style usage is to always capitalize Inline Hooks and names of specific ones like Registration Inline Hook.

Reply:
Thanks. Will review all topics to ensure style guidelines.